egs/ami/s5/local/tensorflow/prep_data.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+#set -v
+set -e
+
+train_text=data/ihm/train/text
+nwords=9999
+
+. path.sh
+. cmd.sh
+
+. utils/parse_options.sh
+
+if [ $# != 1 ]; then
+   echo "Usage: $0 [options] <dest-dir>"
```

Comment:
There needs to be an explanation of what this script is for and what it does.  The same goes for any other scripts that you are adding.